datanommer.models/datanommer/models/__init__.py

```diff
@@ -394,7 +394,9 @@ def grep(
 
         if contains:
             query = query.filter(
-                or_(*(Message._msg.like("%%%s%%" % contain) for contain in contains))
+                or_(
+                    *(Message.msg.op("#>>")("{}").like(contain) for contain in contains)
```

Comment:
So, we're not wrapping contain with percent signs anymore, should we restore that?